RELEASE.md

```diff
@@ -22,6 +22,8 @@
 
 10. Add the release to `elixir.csv` and `_data/elixir-versions.yml` files in `elixir-lang/elixir-lang.github.com`
 
+11. Send an e-mail to elixir-lang-ann@googlegroups.com with title "Elixir vVERSION released" with a link to the GitHub Releases page. If it is a security release, include `[security]` in the title
```

Comment:
Minor nitpick to clarify the email:

> Send an e-mail to elixir-lang-ann@googlegroups.com with title "Elixir vVERSION released". The body should should be/contain a link to the [GitHub Releases](https://github.com/elixir-lang/elixir/releases) page. If it is a security release, include `[security]` in the title

Note "be/contain", because I don't know if the body should be just the link. I guess it should be "be".

I've also included a link to https://github.com/elixir-lang/elixir/releases. 

And maybe specify in which part should the "tag" `[security]` be placed? A few options:

> If it is a security release, prefix the title with the `[security]` tag
> If it is a security release, prepend the `[security]` tag to the title
> If it is a security release, suffix the title with the `[security]` tag
> If it is a security release, append the `[security]` tag to the title